api/v1/awsloadbalancercontroller_types.go

```diff
@@ -162,6 +170,19 @@ type AWSLoadBalancerDeploymentConfig struct {
 	Replicas int32 `json:"replicas,omitempty"`
 }
 
+// AWSLoadBalancerCredentialsRequestConfig defines customization options for the controller's CredentialsRequest.
+type AWSLoadBalancerCredentialsRequestConfig struct {
+	// stsIAMRoleARN is the Amazon Resource Name (ARN) of an IAM Role
+	// which must be manually created for the controller's CredentialsRequest.
+	// This ARN is added to AWSProviderSpec initiating the creation of a secret containing IAM
+	// Role details necessary for assuming the IAM Role via Amazon's Secure Token Service (STS).
+
```

Comment:
Should remove this blank line, replace with `//` if you want a break

Reply:
Done.